rpcq/_base.py

```diff
@@ -13,16 +13,21 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 ##############################################################################
+
 import rapidjson
-from copy import deepcopy
-from ruamel import yaml
 
 import msgpack
+from dataclasses import astuple, replace, fields, MISSING
+from ruamel import yaml
 
 REPR_LIST_TRUNCATION = 10
 "Number of list elements to print when calling repr on a Message with a list field."
 
 
+global message_types
```

Comment:
Not a fan of `.__subclasses__()`?

Reply:
I didn't know that was a thing! Of course I prefer that.